### PR TITLE
Add title service

### PR DESCRIPTION
**Pull request by YehorPytomets**
### Changes introduced by this PR:
* Add `TitleService` to allow dynamically change page title.
* Make `TitleService` a singleton class in order to reach a single instance of it from anywhere in the app.

2 Apr 2020 @YehorPytomets : PR is ready to review.

### CODE REVIEW

File: webclient/app/services/services.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  /**
+   * @type {TitleService}.
+   */
+  titleService;
+
+  /**
+   * Instantiates services.
+   */
+  constructor() {
+    if (!this.titleService) {
```
Comment (dkalinin) [inline webclient/app/services/services.js:19]: This is always false.

File: webclient/tests/app/services/title-service-tests.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    this._title = title;
+  }
+
+  get title() {
+    return this._title;
+  }
+
+  set title(title) {
+    this._title = title;
+  }
```
Comment (dkalinin) [inline webclient/tests/app/services/title-service-tests.js:57]: These getters and setters are redundant.

File: webclient/app/services/services.js (hunk 1 above)
Comment (YehorPytomets) [reply webclient/app/services/services.js]: Done.

File: webclient/tests/app/services/title-service-tests.js (hunk 2 above)
Comment (YehorPytomets) [reply webclient/tests/app/services/title-service-tests.js]: Done.

**Pull request merged**